Infrastructure.DataAccess/Mapping/OptionEntityMap.cs

```diff
@@ -8,6 +8,7 @@ public abstract class OptionEntityMap<T, TReference> : EntityMap<T>
         protected OptionEntityMap()
         {
             this.Property(t => t.Name)
+                .HasMaxLength(OptionEntity<TReference>.MaxNameLength)
```

Comment:
kan vi tillade os det med db i dag?

Reply:
Nej der er en enkelt ItContractType som er på 109. Ellers var de andre høje omkring de 60. Så jeg hæver den lige til 150 i stedet for 100.

Ligner også det er en ret unormal ItContractType: 
![image](https://user-images.githubusercontent.com/39791946/114832067-5cc26680-9dce-11eb-95ab-59abf4efe8eb.png)

Reply:
jøsses det er da også en lang en